Model/Sync/Orders/Processor.php

```diff
@@ -202,6 +206,18 @@ public function processOrders($orderItems = [])
 
         $ordersWithMissedProducts = [];
         foreach ($orderItems as $order) {
+            $storeId = $order->getStoreId();
+            if ($this->config->syncResetInProgress($storeId, 'order')) {
+                $this->yotpoOrdersLogger->info(
+                    __(
+                        'Order sync is skipped because order sync reset is in progress
+                        - Magento Store ID: %1, Name: %2',
+                        $storeId,
+                        $this->config->getStoreName($storeId)
+                    )
+                );
+                continue;
+            }
```

Comment:
Why do we need this if we've returned the method earlier (lines 166-169)?

Reply:
Sync process is taking a batch of 100 records and then processing them in the loop. So if it is in between a loop of 100 records, we should skip that loop iterations also if there is a reset sync happened. That is why we needed this lines here.

Reply:
But if the sync already started, and it came to this point while reset sync was executed in the meantime, it means that all of the orders in this iteration should still be synced.
I'm still not convinced why we need this here.

Reply:
Hi @LiorGingi ,

Are you saying that the orders should be synced regardless of whether the reset sync was executed in between a batch of sync? 

I guess it shouldn't sync in the current iteration because it will try to sync to the wrong yotpo account if that is the case. Admin would hit the reset sync command once after he changes the Yotpo account Id. In this case, all the current sync iterations should be stopped ASAP. That is why we put the condition in 2 places.

Reply:
We will remove this from here.